modules/nf-core/happy/happy/main.nf

```diff
@@ -9,29 +9,49 @@ process HAPPY_HAPPY {
         'quay.io/biocontainers/hap.py:0.3.14--py27h5c5a3ab_0' }"
 
     input:
-    tuple val(meta), path(truth_vcf), path(query_vcf), path(bed)
-    tuple path(fasta), path(fasta_fai)
+    tuple val(meta), path(query_vcf), path(truth_vcf), path(regions_bed), path(targets_bed)
```

Comment:
iirc the truth vcf, regions and target bed are not sample specific. the truth vcf could be compared against various own vcfs, no?

Reply:
fair enough. I was thinkinf of use cases were you benchmark a bunch of vc of one sample for example. But this should cover both then, so good for me